AwsCryptographicMaterialProviders/dafny/AwsCryptographyKeyStore/src/ErrorMessages.dfy

```diff
@@ -0,0 +1,39 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+module {:options "/functionSyntax:4" } KeyStoreErrorMessages {
+  const KMS_CONFIG_KMS_ARN_INVALID :=
+    "KMSConfiguration's KMS Key ARN is invalid."
+
+  const CUSTOM_BRANCH_KEY_ID_NEED_EC :=
+    "Custom branch key id requires custom encryption context."
+
+  const GET_KEY_ARN_DISAGREEMENT :=
+    "Branch Key's AWS KMS Key ARN read from Dynamodb does not match Key Store's configured AWS KMS Key ARN."
```

Comment:
nit: consistency - we don't use AWS KMS, just KMS elsewhere

Reply:
Resolved in 6bac41f3